db/migrate/20151125032708_create_visits.rb

```diff
@@ -2,6 +2,7 @@ class CreateVisits < ActiveRecord::Migration
 
   def change
     create_table :visits do |t|
+      t.references :commentable, polymorphic: true, index: true
```

Comment:
Why does the `Visit` model have a `commentable` reference?
Shouldn't the `Comment` model have a `commentable` reference?

If you look in the schema, a `commentable` reference creates two column on the model that has it, namely `commentable_id` and `commentable_type`. So if a comment was made on a `Student`, then the `commentable_id` would be that student's `id` and the `commentable_type` would be `'Student'`.

Let me know if you have any questions about that! Following those changes, the `Comment` model should have a declaration `belongs_to :commentable, polymorphic: true` and then both the `School` and `Student` models should have a `has_many :comments`.